tests/foreman/performance/test_standard_prep.py

```diff
@@ -83,17 +81,47 @@ def _download_manifest(self):
         self.logger.info('Downloading manifest complete.')
 
     def _upload_manifest(self):
+        self.org_id = self._get_organization_id()
+        self.logger.debug('org-id is {}'.format(self.org_id))
+
         result = Subscription.upload({
-            'file': '/root/{}'.format(self.manifest_file),
+            'file': '/root/{}'.format(MANIFEST_FILE_NAME),
             'organization-id': self.org_id
         })
 
         if result.return_code != 0:
             self.logger.error('Fail to upload manifest!')
             return
-        (self.sub_id, self.sub_name) = self._get_subscription_id()
         self.logger.info('Upload successful!')
 
+        # after uploading manifest, get all default parameters
+        self.pid = self._get_production_id()
+        (self.sub_id, self.sub_name) = self._get_subscription_id()
+        self.logger.debug('product id is {}'.format(self.pid))
+
+    def _get_organization_id(self):
+        """Get organization id"""
+        result = Org.list(per_page=False)
+        if result.return_code != 0:
+            self.logger.error('Fail to list default organization.')
+            return
+        return result.stdout[0]['id']
+
+    def _get_production_id(self):
+        """Get organization id"""
+        result = Product.list(
+            {'organization-id': self.org_id},
+            per_page=False
+        )
+
+        if result.return_code != 0:
+            self.logger.error('Fail to list default products.')
+            return
+        return [result.stdout[i]['id']
+                for i in range(len(result.stdout))
+                if result.stdout[i]['name'] ==
+                u'Red Hat Enterprise Linux Server'][0]
```

Comment:
Please indent your code instead of using hanging blocks:

```python
        return [
            result.stdout[i]['id']
            for i in range(len(result.stdout))
            if result.stdout[i]['name'] == u'Red Hat Enterprise Linux Server'
        ][0]
```

While I'm at it, though - is there any reason to compile a list of results and
then return the 0th result? I think this is equivalent:

```python
        for item in result.stdout:
            if item['name'] == u'Red Hat Enterprise Linux Server'
                return item
```